generated/ruby/google-cloud-ruby/google-cloud-dlp/google-cloud-privacy-dlp.gemspec

```diff
@@ -0,0 +1,29 @@
+# -*- ruby -*-
+# encoding: utf-8
+
+Gem::Specification.new do |gem|
+  gem.name          = "google-cloud-privacy-dlp"
+  gem.version       = "0.6.8"
```

Comment:
I think we can configure the generated package version for ruby in artman_dlp.yaml just like

https://github.com/googleapis/googleapis/blob/master/gapic/api/artman_dlp.yaml#L48

Similar to video intelligence, the start version can be '0.20.0'.